nx.json

```diff
@@ -1,15 +1,30 @@
 {
-  "npmScope": "spartacus",
-  "tasksRunnerOptions": {
-    "default": {
-      "runner": "nx/tasks-runners/default",
-      "options": {
-        "cacheableOperations": ["build", "lint", "test", "e2e"]
```

Comment:
[QUESTION]
Previously it contained `e2e`. In the new version of this file, I can't find it. Was it deliberate?